Overview.src.html

```diff
@@ -3433,12 +3501,19 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
    <dt><code title>USVString</code>
    <dd>
-    <p>Push the result of running <span data-anolis-spec=encoding>utf-8 encode</span> on
-    <var>object</var> to <var>stream</var>.
+    <p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> object
+    wrapping an <code>ArrayBuffer</code> containing the result of running
+    <span data-anolis-spec=encoding>utf-8 encode</span> on <var>object</var> to <var>stream</var>.
```

Comment:
Ah yes, that cannot fail. The utf-8 encoder accepts all Unicode scalar values. There are no errors. (Implementations might have error situations that in the specification are handled through the USVString annotation. Depends on how you implement things.)